examples/job/run-simulations-and-extract-properties.py

```diff
@@ -92,12 +92,12 @@
 
 # #### Workflow
 # 
-# This example is based on [this](https://platform.exabyte.io/analytics/workflows/BEWfDREDFFL9g8Qpk) bank workflow which is later copied to the account workflows collection.
+# This example is based on [this](https://platform.exabyte.io/analytics/workflows/56xDyXsPMNJ7cF9nv) bank workflow which is later copied to the account workflows collection.
```

Comment:
As a note of caution to ourselves in the future, we should add a quick description of this workflow (name, subworkflows, unit names), so we can find a substitute quickly in case the ID gets scrambled again.